terraform/modules/hub/vpc.tf

```diff
@@ -37,7 +37,9 @@ resource "aws_vpc_endpoint" "s3" {
                 "arn:aws:s3:::gds-${var.deployment}-ssm-session-logs-store",
                 "arn:aws:s3:::gds-${var.deployment}-ssm-session-logs-store/*",
                 "arn:aws:s3:::govukverify-self-service-${var.deployment}-config-metadata",
-                "arn:aws:s3:::govukverify-self-service-${var.deployment}-config-metadata/*"
+                "arn:aws:s3:::govukverify-self-service-${var.deployment}-config-metadata/*",
+                "arn:aws:s3:::govukverify-self-service-integration-config-metadata",
```

Comment:
this also whitelist the S3 integration bucket in all environments 🤔

Reply:
We will address this again when we come to move the S3 bucket into it's own module in the hub